Workbooks/Workloads/Alerts/Alerts.workbook

```diff
@@ -736,7 +736,7 @@
           {
             "type": 1,
             "content": {
-              "json": "Alert templates and the rules based on them are scoped to the monitoring data from the Monitoring profile you are viewing. Select the 'Create' link and you will be prompted to enter the threshold and other details needed to create an Alert rule.\r\n\r\nPlease be aware that enabling alerts will impact the cost of managing your resource. [Learn more about the pricing structure](https://aka.ms/ci-alert-pricing)"
+              "json": "Alert templates and the rules based on them are scoped to the monitoring data from the Monitoring profile you are viewing. Select the 'Create' link and you will be prompted to enter the threshold and other details needed to create an Alert rule.\r\n\r\nPlease be aware that enabling alerts will impact the cost of managing your resource. [Learn more about the pricing structure](https://aka.ms/ci-alert-pricing)<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 16 16\" height=\"12\" width=\"12\" style=\"margin: 0px 0px -2px 3px\">\r\n  <path d=\"M14.3,1.5H10V.5h6v6H15V2.2l-8,8-.7-.7Z\" />\r\n  <path d=\"M11,8.5v5a.9.9,0,0,1-.787,1,.948.948,0,0,1-.213,0H2a.9.9,0,0,1-1-.787A.948.948,0,0,1,1,13.5v-7a.9.9,0,0,1,.787-1A.948.948,0,0,1,2,5.5H8l1-1H2a2,2,0,0,0-2,2v7a2,2,0,0,0,2,2h8a2,2,0,0,0,2-2v-6Z\" />\r\n</svg>"
```

Comment:
since the icon is being used in multiple places, maybe have an icon image would be better if you can do it

Reply:
yeah that's true, but if I host the image somewhere, there's the problem of maybe some clouds can't get access to the image